tag/map_go17.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build go1.6,go1.7
```

Comment:
Can we just depend on this everywhere. The stdlib sort is implementing the same.

Reply:
What do you mean by "we just depend on this everywhere"? Some of the code needs features that are in Go1.8 such as `runtime.CallersFrames` while others for code in Go1.7.